README.md

```diff
@@ -35,6 +35,7 @@ For each account the following metrics are exported:
  * *stellar_account_selling_liabilities*
  * *stellar_account_num_sponsored*
  * *stellar_account_num_sponsoring*
+ * *stellar_account_available_balance*
```

Comment:
Maybe put it below `stellar_account_balance` so that they are next to each other?

Reply:
https://github.com/stellar/stellar-account-prometheus-exporter/pull/11/commits/0a96d61445767abf2db2b888bb0db38c7913bd46